assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.js.es6

```diff
@@ -0,0 +1,13 @@
+export default {
+  setupComponent(args, component) {
+    const enabled = Discourse.SiteSettings.location_users_map;
+    if (enabled) {
+      Ember.run.scheduleOnce('afterRender', () => {
+        const $existingLocation = $('span.location');
+        component.$('span.map-location').insertAfter($existingLocation);
+        $existingLocation.hide();
+        $('span.website-name').css("margin-top","-3%");
```

Comment:
I can see the problem you're trying to solve here, but this is not a good solution. It won't work if the map location is only takes up one line. For example "Australia" is a valid geo-location. This is what it looks like:

<img width="259" alt="screenshot at sep 25 19-26-34" src="https://user-images.githubusercontent.com/5931623/46005615-fe8ee480-c0f8-11e8-9277-02a70fe29660.png">

What you need is to make use of the flex-box of the parent element. The reason the elements are centre-aligned is because the flex-box has ``align-items: centre;``. You need to update that instead.

Reply:
Fixed by this ab700c252794a683e9bde2120bdb179cee6ca25d